routes/web.php

```diff
@@ -22,11 +22,13 @@
     });
     Route::middleware(['checklogout'])->group(function () {  
         //  
-        Route::view('/login', 'login')->name('loginpage');
+        Route::get('/login',[UserController::class,'getLoginView'])->name('loginpage');
+        Route::get('/register',[UserController::class,'getRegisterView'])->name('register_page');
     });
     //====================get==========================
     Route::get('change-language/{language}', [SetLangController::class,'setLang'])->name('user.change-language');
```

Comment:
name route đặt theo rạng snake_case
```suggestion
    Route::get('change-language/{language}', [SetLangController::class,'setLang'])->name('user.change_language');
```

Reply:
ok a